tests/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/icinga/icingadb/tests
 
 go 1.16
 
+replace github.com/icinga/icinga-testing => github.com/icinga/icinga-testing v0.0.0-20220513144038-4aef51dce82a
```

Comment:
I just approved and merged https://github.com/Icinga/icinga-testing/pull/24, so just go ahead and just do a regular update instead.